### PR TITLE
Upgraded to sbt-scoverage 1.5.0 and provided two full working examples.

### PR DESCRIPTION
**Pull request by dblock**
Discussion in https://github.com/scoverage/sbt-scoverage/issues/202#issuecomment-259670217.

### CODE REVIEW

**Comment by dblock** [comment]
Don't merge until we get all comments from @gslowikowski in, he understands sbt-scoverage a lot better than me :)

**Comment by dblock** [comment]
I think this is good to merge. It's definitely more accurate that what I PRed last time, we're running with what's here, so go for it.

File: jekyll/_docs/code-coverage.md [hunk 1]
```diff
@@ -52,35 +52,39 @@ The [simplecov README](https://github.com/colszowka/simplecov/#getting-started)
...
-    - echo "Skipping sbt test:compile, will compile during test coverage."
-test:
-  override:
-    - sbt coverage test
-    - sbt 'set test in assembly := {}' assembly
+    - sbt coverage test:compile
 ```
 
-Produce the coverage report and gather coverage artifacts.
+Run tests with coverage with `sbt coverage test` and produce a coverage report with `sbt coverageReport`.
```
Comment (smart-alek) [inline jekyll/_docs/code-coverage.md]: Remove "with coverage"; we're already operating in the context of that tool.

"Run tests with `sbt coverage test` and produce a coverage report with `sbt coverageReport`."

File: jekyll/_docs/code-coverage.md [hunk 2]
```diff
@@ -52,35 +52,39 @@ The [simplecov README](https://github.com/colszowka/simplecov/#getting-started)
...
-    - find target -name scoverage-report -exec cp -r {} $CIRCLE_ARTIFACTS \;
+  post:
+    - mkdir -p $CIRCLE_ARTIFACTS/scala-2.10
+    - mv target/scala-2.10/coverage-report  $CIRCLE_ARTIFACTS/scala-2.10/coverage-report
+    - mv target/scala-2.10/scoverage-report $CIRCLE_ARTIFACTS/scala-2.10/scoverage-report
 ```
 
+See [gslowikowski/sbt-simple-circleci-test](https://github.com/gslowikowski/sbt-simple-circleci-test) for a complete example.
+
+For multimodule projects run tests against multiple Scala versions with `sbt coverage +test` and aggregate reports with `sbt coverageAggregate`. See [gslowikowski/sbt-multimodule-circleci-test](https://github.com/gslowikowski/sbt-multimodule-circleci-test) for a complete example.
```
Comment (smart-alek) [inline jekyll/_docs/code-coverage.md]: Add comma:

"For multimodule projects, run tests against multiple Scala versions with..."

**Comment by dblock** [comment]
Nitpicks addressed, thanks @smart-alek!

**Pull request merged**